TestModule/CountFilesByTypeInDirectory.cpp

```diff
@@ -1,9 +1,8 @@
 #include "CountFilesByTypeInDirectory.h"
+#include "../srcStatic/FileSystemHelper.h"
```

Comment:
There should already be an `AdditionalIncludeDirectories` setting for `../srcStatic/`. We should use that over relative include paths between the projects.